.github/workflows/pkgcheck.yml

```diff
@@ -28,4 +28,9 @@ jobs:
         sudo ln -s /var/db/repos/gentoo/profiles/default/linux/amd64/17.1 /etc/portage/make.profile
     - name: Test with pkgcheck
       run: |
-        pkgcheck scan --exit --checks=-RedundantVersionCheck --keywords=-MissingAccountIdentifier
+        # There is a bug open with pkgcheck to fix the false positives with
+        # NonsolvableDepsInXXX
+        #
+        # dev-haskell/megadeth and dev-haskell/ttasm are masked for removal.
```

Comment:
Meanwhile we can also set `KEYWORDS=""` on masked ebuilds to avoid silencing the check.

Reply:
I was thinking the same, but wasn't sure if it'd be redundant due to also masking the packages.

I'll make the change